### ISSUE TITLE
How to put scalars in the same row in tensorboard in TF2.0?

### ISSUE THREAD

**Issue by zhangjh915** [tensorflow/tensorflow#34668]
<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>

**System information**
- TensorFlow version (you are using): 2.0
- Are you willing to contribute it (Yes/No): Yes

**Describe the feature and the current behavior/state.**

I did not see related arguments in the documentation. If there is such function, please let me know. :P

The following code produces a tensorboard with 3 different rows. How to put the first two in one row and the second in the other?

```
with summary_writer.as_default():
    tf.summary.scalar("training_loss", loss, step=batch_index)
    tf.summary.scalar("learning_rate", optimizer.learning_rate, step=batch_index)
with summary_writer.as_default():
    tf.summary.scalar("validation_loss", val_loss, step=batch_index)
```

**Will this change the current api? How?**
Probably
**Who will benefit with this feature?**

**Any Other info.**

**Comment by rmothukuru** [comment]
@zhangjh915,
Can you please refer the issue, [7089 ](https://github.com/tensorflow/tensorflow/issues/7089) and this [Stack Overflow Issue](https://stackoverflow.com/questions/48951136/plot-multiple-graphs-in-one-plot-using-tensorboard/48951430) and let us know if it helps. Thanks!

**Comment by zhangjh915** [comment]
@rmothukuru ,
Thanks for the quick reply.  I read the solutions in the links you attached but they both seem to not be TF2.0. To my knowledge, there is neither "session.run" or "add_scalars" in TF2.0. To make my question clear, The the code I attached in the question produced the following tensorboard. But I want to put both learning rate and training loss into the same row.
![截屏2019-11-2915 03 38](https://user-images.githubusercontent.com/38232357/69850278-e47f0200-12b9-11ea-916e-daf551620a01.png)

**Comment by rmothukuru** [comment]
@zhangjh915,
Please refer this [Tensorboard Link](https://www.tensorflow.org/tensorboard/migrate) to migrate 1.x code to 2.0. Thanks!

**Comment by zhangjh915** [comment]
@rmothukuru ,
Sorry but it does not help with my question. The solutions you attached is to plot different curves in the same graph, but what I want is to put different graphs in the same row of category. In the picture I attached above, I want to put the first two plots in the same row and set the row name to something like "training".
